### DISCUSSION TITLE
Ask a question

### DISCUSSION THREAD

**Discussion by diditpribadi** [FlowmapBlue/FlowmapBlue#37] [Q&A] [answered]
Dear @ilyabo I just have a question. When I use flowmapblue R package I have put my map token but the base map does not appear. The picture appears in viewer only the arrow and circle of origin and destination. Could you help me with this issue?

**Comment by ilyabo** [comment, marked as answer]
Hi @diditpribadi! Yes, there seems to be an issue in R Studio. For some reason, the Mapbox map layer isn't rendered in the embedded Chromium browser when loaded in R Studio. However, if you open it in a separate window (I think there was a button for that in R Studio) or export the map, the background map should be visible.
